pkg/shell/plot.js

```diff
@@ -352,7 +353,13 @@ shell.plot = function plot(element, x_range_seconds, x_stop_seconds) {
                                             limit: (x_offset > 0) ? (x_range / interval) : undefined,
                                             host: desc.host
                                           });
-        $(archive_channel).on("message", handle_message);
+        $(archive_channel).on("message", function(data) {
+            if (first) {
+                first = false;
+                $("#dashboard-toolbar").show();
```

Comment:
This will only show the toolbar if there are archived samples for the last five minutes, but not if you have archived samples, but only until half an hour ago, say.